templates/monitoring/monitoring.nginx.j2

```diff
@@ -13,8 +13,7 @@ location /mon/bbb/ {
   include proxy_params;
 }
 {% else %}
-{% set bbb_monitoring_exporter_port = (bbb_monitoring_all_in_one_enabled | bool
-    | ternary(bbb_monitoring_all_in_one_port, bbb_monitoring_systemd_port, 9688)) %}
+{% set bbb_monitoring_exporter_port = (bbb_monitoring_all_in_one_enabled true | ternary(bbb_monitoring_all_in_one_port, bbb_monitoring_systemd_port, 9688)) %}
```

Comment:
@wilkis3 Thank you for fixing my suggestion!

Reply:
@crosscodr the 'd' of bbb_monitoring_all_in_one_enable**d** was too much.